src/index.ts

```diff
@@ -184,6 +184,12 @@ export default class WebPhone {
             allowLegacyNotifications: true
         };
 
+        if (options.builtinEnabled === null || options.builtinEnabled === undefined) {
+            configuration.log.builtinEnabled = true;
+        } else {
+            configuration.log.builtinEnabled = options.builtinEnabled;
+        }
+
```

Comment:
What was wrong with the original way of passing this down to `log`?

Reply:
'builtinEnabled: options.builtinEnabled || true' will always return true, so user cannot disable log with options.builtinEnabled

Reply:
Right. Good catch. It should be `typeof options.builtinEnabled === 'undefined' ? options.builtinEnabled : true`.